odoo/tools/translate.py

```diff
@@ -603,7 +603,7 @@ def __iter__(self):
             translation = entry.msgstr
             found_code_occurrence = False
             for occurrence, line_number in entry.occurrences:
-                match = re.match(r'(model|model_terms):([\w.]+),([\w]+):(\w+)\.(\w+)', occurrence)
```

Comment:
@jva-odoo @mja-odoo Why/Where do we have xml_ids with -?

Reply:
@jco-odoo https://github.com/odoo/odoo/blob/master/addons/l10n_be/data/account_tax_template_data.xml#L4

Reply:
Done at 6d98f838571964835e77208cacfc628786cb01

Reply:
@mart-e Ok 👍 
This is a temporary fix in this brach so now we remove it.
Thank you